uitest/README.md

```diff
@@ -0,0 +1,44 @@
+## Dev Home UI Testing
+### How to run UI tests?
```

Comment:
Nit: I'd say either "How do I run UI tests?" or "How to run UI tests" (with no question mark)